app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -44,21 +45,32 @@ function AddressSearch({
   const ref = useRef<SpinnerButtonRefHandle>(null);
 
   useEffect(() => {
-    if (addressCandidates) {
+    if (addressCandidates && addressCandidates.length > 0) {
+      setNoAddressFoundErrors('');
+      validatedFieldRef.current?.setCustomValidity('');
+      validatedFieldRef.current?.reportValidity();
       const bestMatchedAddress = addressCandidates[0];
       onAddressFound(bestMatchedAddress);
       ref.current?.toggleSpinner(false);
     }
-  }, [addressCandidates]);
+    if (addressCandidates?.length === 0) {
+      setNoAddressFoundErrors('Not a real address');
```

Comment:
Is this state being used for anything? If yes, I'd also the text to be translated.

Reply:
I removed the state variable.

Reply:
fixed in a1e9e956d86cd0321c0a11dde5fd4a6e9851e00d